libtcmu_log.h

```diff
@@ -22,7 +22,8 @@
 #define TCMU_LOG_DEBUG_SCSI_CMD	(LOG_DEBUG + 1)	/* scsi cmd debug-level messages */
 
 /* default tcmu log dir path */
-#define TCMU_LOG_DIR_DEFAULT   "/var/log"
+#define TCMU_LOG_DIR_DEFAULT	"/var/log"
+#define TCMU_CONFIGFILE_DEFAULT	"/etc/tcmu/tcmu.conf"
```

Comment:
I think this should have gone to libtcmu_config.h.

I will just fix this up when I merge the set.

Reply:
I cleaned this up here:

https://github.com/open-iscsi/tcmu-runner/pull/509